Marlin/Marlin_main.cpp

```diff
@@ -4159,7 +4146,7 @@ inline void gcode_M42() {
     float  X_current = current_position[X_AXIS],
            Y_current = current_position[Y_AXIS];
 
-    #if ENABLED(Z_PROBE_SLED) || ENABLED(Z_PROBE_ALLEN_KEY)
+    #if ENABLED(Z_PROBE_ALLEN_KEY)
```

Comment:
A lot of moves are done - but it works.
With the same system - storing x/y where the deploy/stowe is initialized an returning to that, even for Allen key probes this should be possible. I suppose it can be made working for any kind of probe.

Currently playing with Allen key probes for Cartesianisch printers.
No , sled probes do not work for deltas - no axis where you could mount the sled.